src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -469,21 +567,29 @@ private JLabel getRowCounter() {
 	private Component getDateFilterPanel() {
 		if (dateFilterPanel == null) {
 			dateFilterPanel = new JPanel(new SpringLayout());
+			
+			dateFrom = new GoodDateChooser(LocalDate.now());
+			dateTo = new GoodDateChooser(LocalDate.now());
+			
+			resetDates();
 			dateFilterPanel.add(new JLabel(MessageBundle.getMessage("angal.common.datefrom.label")));
-			if (!GeneralData.ENHANCEDSEARCH) {
-				dateFrom = new GoodDateChooser(LocalDate.now().minusWeeks(1));
-			} else {
-				dateFrom = new GoodDateChooser(LocalDate.now());
-			}
 			dateFilterPanel.add(dateFrom);
 			dateFilterPanel.add(new JLabel(MessageBundle.getMessage("angal.common.dateto.label")));
-			dateTo = new GoodDateChooser(LocalDate.now());
 			dateFilterPanel.add(dateTo);
 			SpringUtilities.makeCompactGrid(dateFilterPanel, 2, 2, 5, 5, 5, 5);
 		}
 		return dateFilterPanel;
 	}
 
+	private void resetDates() {
+		if (!GeneralData.ENHANCEDSEARCH) {
+			dateFrom.setDate(LocalDate.now().minusWeeks(1));
+		} else {
+			dateFrom.setDate(LocalDate.now());
```

Comment:
```suggestion
			dateFrom.setDate(TimeTools.getNow()));
```

We had decided to replace `LocalDate.now()` with '`TimeTools.getNow()` didn't we?

Reply:
yep! Thanks!